interfaces/acados_matlab_octave/AcadosMultiphaseOcp.m

```diff
@@ -106,6 +108,9 @@
             acados_folder = getenv('ACADOS_INSTALL_DIR');
             obj.acados_include_path = [acados_folder, '/include'];
             obj.acados_lib_path = [acados_folder, '/lib'];
+
+            obj.casadi_pool_names = [];
```

Comment:
I think we dont have this property in the corresponding python class. Is it needed here? why?

Reply:
Oh I see, we do set it, but we just didn't add this property and just write it to the class anyway.
Maybe we should make it a private property in python then.